channels/drive/client/drive_main.c

```diff
@@ -64,6 +64,8 @@ struct _DRIVE_DEVICE
 	rdpContext* rdpcontext;
 };
 
+static UINT sys_code_page = 0;
```

Comment:
@hardening We should do another pr for that. Remove all instances of hard coded ```CP_UTF8``` with ```GetACP()``` calls.